chalk-integration/src/lowering.rs

```diff
@@ -1074,6 +1074,17 @@ impl LowerTy for Ty {
                 };
                 Ok(chalk_ir::TyData::Function(function).intern(interner))
             }
+            Ty::Tuple { arity } => Ok(chalk_ir::TyData::Apply(chalk_ir::ApplicationTy {
+                name: chalk_ir::TypeName::Tuple(arity),
+                substitution: chalk_ir::Substitution::empty(interner),
```

Comment:
`chalk_ir::Substition::from_fallible(interner, types.iter().map(...)) // as in the code above`